server/neptune/gateway/event/push_event_handler.go

```diff
@@ -41,7 +42,7 @@ type deploySignaler interface {
 }
 
 type rootConfigBuilder interface {
-	Build(ctx context.Context, repo models.Repo, branch string, sha string, installationToken int64) ([]*valid.MergedProjectCfg, error)
+	Build(ctx context.Context, repo models.Repo, branch string, sha string, fileFetcherOptions github.FileFetcherOptions, installationToken int64) ([]*valid.MergedProjectCfg, error)
```

Comment:
Not a blocker but we use `fileFetcherOptions` as variadic argument and imo we should follow go's best practice of having variadic args at the end of the params for readability.  Then, you can just create the  `FileFetcherOptions` object in the call instead of having to create a variable and pass it into the function. What do you think?